src/controls/calendar/EventDetailsPopover.tsx

```diff
@@ -0,0 +1,167 @@
+import * as React from 'react';
+
+import {
+  AvatarGroup,
+  AvatarGroupItem,
+  AvatarGroupPopover,
+  Badge,
+  Body1,
+  Button,
+  Divider,
+  Subtitle1,
+  partitionAvatarGroupItems,
+} from '@fluentui/react-components';
+import { Card, RenderLabel, Stack } from '@nuvemerudita/react-controls';
+
+import { IEvent } from './models/IEvents';
+import { css } from '@emotion/css';
+import strings from 'ControlStrings';
+import { useUtils } from './hooks/useUtils';
+
+export interface IEventDetailsPopoverProps {
+  event: IEvent;
+}
+
+const PADDING_LEFT = '32px';
```

Comment:
If you make the constant as just `32` you can use it in `fieldContainer` as well